packages/dialtone-docs/src/content/reference/reference-component-api-patterns.md

```
@@ -143,11 +143,13 @@ Icon scale values correspond to pixel sizes defined in `packages/dialtone-icons/
 Group-child relationships use Vue's provide/inject. The parent provides a reactive context object plus action methods:
 
 **Tab group** (`tab_group.vue`):
+
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)